scripts/ci-k8s-unit-test.sh

```diff
@@ -216,9 +219,15 @@ then
     fi
 
     run_remote_cmd ${VM_PUB_IP} ${SSH_KEY_FILE} "c:/k8s_unit_windows.ps1 -repoName ${REPO_NAME} -repoOrg ${REPO_OWNER} -pullRequestNo ${PULL_NUMBER} -pullBaseRef ${PULL_BASE_REF} ${test_packages_arg}"
```

Comment:
For example, I think you can add a `if` here and do an `echo` to indicate that an error occurred here but the script will continue.

Not a must have but a nice to have.

Reply:
let me try that

Reply:
I think this worked, PTAL and thanks for the tip!

Reply:
Ditto.